src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -1,213 +1,347 @@
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import { mount } from 'enzyme';
 import moment from 'moment';
+import { renderWithStoreAndRouter } from '../../mocks/setup';
+import userEvent from '@testing-library/user-event';
 
 import {
   DateTimeRequestPage,
   getOptionsByDate,
 } from '../../../new-appointment/components/DateTimeRequestPage';
 
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCancel: true,
+  },
+};
+
+function getMondayTruFriday(date) {
+  if (date.day() === 6) {
+    // Move date to next week Monday
+    return date.day(8);
+  } else if (date.day() === 0) {
+    // Move date to current week Monday
+    return date.day(1);
+  }
+  return date;
+}
+
 describe('VAOS <DateTimeRequestPage>', () => {
-  it('should render', () => {
-    const onCalendarChange = sinon.spy();
-
-    const form = mount(
-      <DateTimeRequestPage
-        data={{ calendarData: {} }}
-        onCalendarChange={onCalendarChange}
-      />,
-    );
-
-    expect(form.find('CalendarWidget').length).to.equal(1);
-    expect(form.find('FormButtons').length).to.equal(1);
-    form.unmount();
+  it('should return options for date with getOptionsByDate', () => {
```

Comment:
I think this code should get covered in other tests, so you could probably remove this test